app/controllers/comments_controller.rb

```diff
@@ -12,8 +12,10 @@ def create
     )
     @new_comment.post_id = @post.id
     if @new_comment.save
+      flash[:notice] = 'Post created successfully.'
       redirect_to "/users/#{@post.author_id}/posts/#{@post.id}", notice: 'Success!'
     else
+      flash[:notice] = 'Post creation failed. Try again'
       render :new, alert: 'Error occured!'
```

Comment:
- [ ] The notice should be `Comment created succefully` instead of `Post ...`. Also, After creating the comment we are getting message `Success!`. Instead we want to see the message `Comment created successfully`. For this I think you have to remove the `notice: 'Success!'` from the line 16.